vm/parser.ts

```diff
@@ -636,7 +636,7 @@ module J2ME {
               r = classInfo.getMethodByName(name, type);
             }
             if (!r) {
-              throw $.newRuntimeException(classInfo.getClassNameSlow() + "." + fromUTF8(name) + "." + fromUTF8(type) + " not found");
+              throw classInfo.getClassNameSlow() + "." + fromUTF8(name) + "." + fromUTF8(type) + " not found";
```

Comment:
How about we throw a JS object (maybe JavaRuntimeException???) and then in translate exception we'll convert that into a proper java exception. This way we don't have to introduce the try/catch's you added in int.ts.